ui/src/app/views/app/component.spec.ts

```diff
@@ -1,8 +1,10 @@
 import { TestBed, async } from '@angular/core/testing';
-import {Title} from '@angular/platform-browser';
+import { Title } from '@angular/platform-browser';
```

Comment:
I think we're just not linting these spec files yet. I've updated the content to be consistent...